client/src/keycloak.js

```diff
@@ -0,0 +1,9 @@
+import Keycloak from 'keycloak-js';
+
+const keycloak = new Keycloak({
+  realm: process.env.REACT_APP_KEYCLOAK_REALM,
```

Comment:
See previous comment. We could use the original names (`KEYCLOAK_REALM`, `KEYCLOAK_AUTH_URL`, `KEYCLOAK_FE_CLIENTID`) here.